components/merino_client/src/merino_client.udl

```diff
@@ -0,0 +1,38 @@
+namespace merino_client {
```

Comment:
The UniFFI book has a [section](https://mozilla.github.io/uniffi-rs/udl_file_spec.html) explaining the UDL file and its syntax.